File: progs/2010-05-26/correr.py
import os

E_min = 10
E_max = 20

for i in range(1, 100, 10):
	#s = r"./wl %d %d&" % (i, i+10)
	#print s
	#os.system(s)
	
	s = "python args.py %d" % i
	os.system(s)

File: progs/2010-05-26/correr.cpp
#include <sstream>
#include <cstdlib>
#include <iostream>

using namespace std;

int main() {
	
	ostringstream nombre;
	
	
	int E_min = 10, E_max = 20;
	nombre << "./wl "  << E_min << " " << E_max << endl; 
	
	system(nombre.str().c_str());
	
}